### ISSUE TITLE
start https://www.nushell.sh  will open multiple instances

### ISSUE THREAD

**Issue by ritulahkar** [nushell/nushell#11880]
### Describe the bug

Simple call to ` start https://www.nushell.sh ` opens many instances of the same web page.
It will give errors as gnome-open not found.

### How to reproduce

start https://www.nushell.sh

will open multiple instances.

### Expected behavior

Expected to open only one instance.

### Screenshots

_No response_

### Configuration

| key                | value                                                                                                                                                                                                                               |
| ------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| version            | 0.90.1                                                                                                                                                                                                                              |
| branch             |                                                                                                                                                                                                                                     |
| commit_hash        | c2992d5d8b8541b64e9fc1440ed2917fc9b757f7                                                                                                                                                                                            |
| build_os           | linux-x86_64                                                                                                                                                                                                                        |
| build_target       | x86_64-unknown-linux-gnu                                                                                                                                                                                                            |
| rust_version       | rustc 1.73.0 (cc66ad468 2023-10-03)                                                                                                                                                                                                 |
| rust_channel       | 1.73.0-x86_64-unknown-linux-gnu                                                                                                                                                                                                     |
| cargo_version      | cargo 1.73.0 (9c4383fb5 2023-08-26)                                                                                                                                                                                                 |
| build_time         | 2024-02-07 02:23:21 +00:00                                                                                                                                                                                                          |
| build_rust_channel | release                                                                                                                                                                                                                             |
| allocator          | mimalloc                                                                                                                                                                                                                            |
| features           | dataframe, default, extra, sqlite, static-link-openssl, trash, which, zip                                                                                                                                                           |
| installed_plugins  | custom-value generate, custom-value generate2, custom-value update, from eml, from ics, from ini, from vcf, gstat, inc, nu-example-1, nu-example-2, nu-example-3, nu-example-config, query, query json, query web, query xml, to qr |

### Additional context

I guess it's because every command from open::commands is run in the start.rs

**Comment by fdncred** [comment]
Must be OS dependent because it works fine on MacOS.

**Comment by ErnestasSku** [comment]
I had the same issue. I initially installed nushell through cargo binstall. This was the cause of a few issues
1. High cpu usage ( 8% cpu usage a single nushell instance)
2. Using start . would open the default file explorer twice (with error)

This is on Ubuntu 22.03 LTS .

After uninstalling it, and reinstalling it through cargo install --locked nu, it fixed the high cpu usage, and double opening of the file explorer. However, I still have the error (even though it opens the explorer correctly)
![image](https://github.com/nushell/nushell/assets/73689469/aa7a361d-8562-4b08-9d44-c4bb780d9591)


Also, I have cargo install-update, which I use to auto update programs, which I think was used to update nu from 0.89 to 0.90.1


**On Windows**
I checked out this on windows as well. I updated it through cargo install-update from 0.74 to 0.91.1 and there is no double opening issue. However I had another issue on windows

![image](https://github.com/nushell/nushell/assets/73689469/33af5b78-0078-4a9a-8409-657d04c32bfa)
